packages/nocodb/src/lib/meta/api/userApi/userApis.ts

```diff
@@ -285,7 +291,8 @@ async function passwordChange(req: Request<any, any>, res): Promise<any> {
   await User.update(user.id, {
     salt,
     password,
-    email: user.email
+    email: user.email,
+    token_expired: true
```

Comment:
What will happen if there are multiple browser sessions (multiple jwt tokens) and one of the users sign-in using new credentials? other tokens will be still valid, right since `token_expired` is set to `true`?